### DISCUSSION TITLE
Using custom icons conditionally

### DISCUSSION THREAD

**Discussion by harshmehta813** [apostrophecms/apostrophe#4512] [Q&A] [answered]
I have set the custom icon for the rich text and image widget, and that is working fine and appearing in every place where it is being used. Now, I do not want the custom icon at some places where it is being used.

For eg. When I have to select a widget inside a column, the custom icon should appear. 
![image](https://github.com/apostrophecms/apostrophe/assets/87087018/abaa4ffb-9d8e-4ba0-89eb-180c4b8cdcde)

But, when I have to add the image in a custom widget, the image icon should not appear.
![image](https://github.com/apostrophecms/apostrophe/assets/87087018/68ac5e5e-d191-4b01-8502-aaabd2f108b2)
(the black logo and image text are coming from the custom image icon)

**Comment by boutell** [comment, marked as answer]
Currently a given widget type can only have one icon.
